reports/results.ipynb

```diff
@@ -47,53 +60,154 @@
+       "      <td>0.488878</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>bpd</td>\n",
+       "      <td>-0.454428</td>\n",
+       "      <td>0.649521</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>depression</td>\n",
+       "      <td>-1.693505</td>\n",
+       "      <td>0.090359</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>EDAnonymous</td>\n",
+       "      <td>1.922717</td>\n",
+       "      <td>0.054516</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>healthanxiety</td>\n",
+       "      <td>0.184029</td>\n",
+       "      <td>0.853990</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>10</th>\n",
+       "      <td>lonely</td>\n",
+       "      <td>2.041229</td>\n",
+       "      <td>0.041228</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>11</th>\n",
+       "      <td>ptsd</td>\n",
+       "      <td>0.551019</td>\n",
+       "      <td>0.581621</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>12</th>\n",
+       "      <td>schizophrenia</td>\n",
+       "      <td>0.246796</td>\n",
+       "      <td>0.805066</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>13</th>\n",
+       "      <td>socialanxiety</td>\n",
+       "      <td>0.990075</td>\n",
+       "      <td>0.322138</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>14</th>\n",
+       "      <td>suicidewatch</td>\n",
+       "      <td>-1.643746</td>\n",
+       "      <td>0.100229</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
+      ],
+      "text/plain": [
+       "   subreddit_topic  test_statistic   p_value\n",
+       "0        addiction       -0.481856  0.629908\n",
+       "1             adhd        2.955800  0.003119\n",
+       "2       alcoholism        1.407757  0.159203\n",
+       "3          anxiety        1.112075  0.266106\n",
+       "4           autism        1.358547  0.174290\n",
+       "5    bipolarreddit       -0.692095  0.488878\n",
+       "6              bpd       -0.454428  0.649521\n",
+       "7       depression       -1.693505  0.090359\n",
+       "8      EDAnonymous        1.922717  0.054516\n",
+       "9    healthanxiety        0.184029  0.853990\n",
+       "10          lonely        2.041229  0.041228\n",
+       "11            ptsd        0.551019  0.581621\n",
+       "12   schizophrenia        0.246796  0.805066\n",
+       "13   socialanxiety        0.990075  0.322138\n",
+       "14    suicidewatch       -1.643746  0.100229"
       ]
      },
+     "execution_count": 38,
      "metadata": {},
-     "execution_count": 5
+     "output_type": "execute_result"
     }
    ],
-   "metadata": {
-    "tags": [
-     "remove-input"
```

Comment:
Just wondering, did we want to show code for this? 
You can add a cell tag to 'remove-input' so that it doesn't display in jupyter book (but not sure if you do actually want to show code)

Reply:
Nah you're right - probably should hide the code..